catalogue/webapp/pages/workv2.js

```diff
@@ -53,7 +56,255 @@ export const WorkPage = ({
   ) || {}).value;
   // We strip the last character as that's what Wellcome Library expect
   const encoreLink = sierraId && `http://search.wellcomelibrary.org/iii/encore/record/C__R${sierraId.substr(0, sierraId.length - 1)}`;
+  if (showRedesign) {
```

Comment:
A thought here would be to create a new component that takes a work and just return that?

Reply:
…thought that might be overkill – I'd be happy enough just to rely on IDE code-folding?

Reply:
The reason would be to continue work on the current version when we need to (say implement something from the new design) without landing up in massive conflict land with git.

Reply:
yeah, fair enough